pkg/destroy/openstack/openstack_deprovision.go

```diff
@@ -541,6 +544,73 @@ func deleteTrunks(opts *clientconfig.ClientOpts, filter Filter, logger logrus.Fi
 	return len(allTrunks) == 0, nil
 }
 
+func deleteLoadBalancers(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
+	logger.Debug("Deleting openstack load balancers")
+	defer logger.Debugf("Exiting deleting openstack load balancers")
+
+	conn, err := clientconfig.NewServiceClient("load-balancer", opts)
+	if err != nil {
+		logger.Fatalf("%v", err)
+		os.Exit(1)
+	}
+
+	newallPages, err := apiversions.List(conn).AllPages()
+	if err != nil {
+		logger.Debug("Unable to list api versions: %v", err)
+	}
+
+	allAPIVersions, err := apiversions.ExtractAPIVersions(newallPages)
+	if err != nil {
+		logger.Debug("Unable to extract api versions: %v", err)
+	}
+
+	var octaviaTagSupport bool
+	octaviaTagSupport = false
+	for _, apiVersion := range allAPIVersions {
+		if apiVersion.ID >= "v2.5" {
```

Comment:
maybe this v2.5 should come from a constant

Reply:
Thanks, Toni.